tools/libfrencutils/mpp_io.c

```diff
@@ -1050,7 +1050,12 @@ void mpp_set_deflation(int fid_in, int fid_out, int user_deflation, int user_shu
     for (int vid_out = 0; vid_out < files[fid_out].nvar; ++vid_out) {
         // if user has not specified options, look up and use settings in input file
         if (user_deflation == -1 || user_shuffle == -1) {
-            mpp_get_varname(fid_in, vid_out, varname);
+            mpp_get_varname(fid_out, vid_out, varname);
+
+            // Don't attempt to the get the deflate level if the variable is not in the input file (i.e lat_bnds,
+            // lon_bnds)
+            if (mpp_var_exist(fid_in, varname) == 0) continue;
+
```

Comment:
If the variable is not in the input file (e.g. lat_bnds) and the user doesn't specify the output settings, which output settings are used?

Reply:
In the case, there wouldn't be any compression for that variable:
```
        double lon_bnds(lon, bnds) ;
                lon_bnds:long_name = "longitude bounds" ;
                lon_bnds:units = "degrees_E" ;
                lon_bnds:axis = "X" ;
                lon_bnds:_Storage = "contiguous" ;
                lon_bnds:_Endianness = "little" ;
```
but the variable is nlonx2, so compressing it will not get you very far.

Reply:
Thanks! sounds good to me